packages/twenty-front/src/pages/settings/Releases.tsx

```diff
@@ -108,7 +111,23 @@ export const Releases = () => {
   }, []);
 
   return (
-    <SubMenuTopBarContainer Icon={IconRocket} title="Releases">
+    <SubMenuTopBarContainer
+      Icon={IconRocket}
+      title="Releases"
+      breadcrumbs={
+        <Breadcrumb
```

Comment:
Couldn't we directly pass the links array as props ? Thus abstracting the Breadcrumb component logic in SubMenuTopBarContainer, this will close the SubMenuTopBarContainer API and avoid passing unwanted components by transclusion.

Reply:
sure, we will work on it 😁